### PR TITLE
month is 0-indexed. link to Date() Constructor.

### PR DESCRIPTION
**Pull request by ctstacey**
> What was wrong/why is this fix needed? (quick summary only)
• clarify why month value 6, in Date() Constructor, turns out to be July (not June).
• add link to Date() Constructor

> Issue number (if there is an associated issue)

> Anything else that could help us review it

### CODE REVIEW

**Comment by ctstacey** [comment]
- not sure how to fix the broken link
- could change "zero" to "0" to shorten line

**Comment by teoli2003** [comment]
I don't think this is useful. The article is about `setDate()` and not `Date()`, and the comment makes it clear that `6` is July.

I think we should only add the link to the constructor in the _See also_ section.

**Comment by ctstacey** [comment]
Fair point. Adding the link in the see also section seems like a good idea!
I still believe a comment saying something like "month is 0-indexed" is useful next to the Date() constructor (in code example).

File: files/en-us/web/javascript/reference/global_objects/date/setdate/index.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ date being set to 1 day before the last day of the previous month.
 ### Using setDate()
 
 ```js
-const theBigDay = new Date(1962, 6, 7, 12); // noon of 1962-07-07 (7th of July 1962)
+const theBigDay = new Date(1962, 6, 7, 12); // noon of 1962-07-07 (7th of July 1962)   *month is zero-indexed
```
Comment (teoli2003) [inline files/en-us/web/javascript/reference/global_objects/date/setdate/index.md:53]:
```suggestion
const theBigDay = new Date(1962, 6, 7, 12); // noon of 1962-07-07 (7th of July 1962,  month is 0-indexed)
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]
And an entry in See also

File: files/en-us/web/javascript/reference/global_objects/date/setdate/index.md [hunk 2]
```diff
@@ -59,6 +59,8 @@ theBigDay.setDate(98);  // 1962-10-06 (6th of October 1962)
 theBigDay.setDate(-50); // 1962-08-11 (11th of August 1962)
 ```
 
+\* month is zero-indexed: [Date() Constructor](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/Date)
```
Comment (teoli2003) [inline files/en-us/web/javascript/reference/global_objects/date/setdate/index.md:62]: To be removed

**Comment by teoli2003** [comment]
To edit a PR, you can click on the _edit file_ menu item:
Screenshot 2021-07-23 at 14 55 25

**Comment by ctstacey** [comment]
Awesome, thank-you.
Just trying to figure out how to turn the link into from:
[Date() Constructor](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/Date)
into a jsxref equivalent
eg.
- {{jsxref("Date.prototype.getDate()")}}

**Comment by teoli2003** [comment]
I would try `{{jsxref("Date.Date()", "Date()")}}`, but to double-check there once committed: https://pr7193.content.dev.mdn.mozit.cloud/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/setDate

**Review by teoli2003 — APPROVED** [review]
Looks good now.

**Comment by ctstacey** [comment]
Awesome teoli2003, thanks for helping, I appreciate it!

**Pull request merged**